test/linalg/symmetric.jl

```diff
@@ -186,6 +191,17 @@ end
                         @test inv(Hermitian(a, uplo))::Hermitian ≈ inv(Matrix(Hermitian(a, uplo)))
                     end
                 end
+                if eltya <: Base.LinAlg.BlasComplex
+                    @testset "inverse edge case with complex Hermitian" begin
```

Comment:
It looks like this testset does not depend on the loop variable(s), so that it would be repeated twice (for the 2 values in `Base.LinAlg.BlasComplex`) with exactly the same code?

Reply:
Did I get this wrong?